iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java

```diff
@@ -99,20 +101,35 @@ public Jobs(final IIoTHubWrapper ioTHubService,
 
     @Override
     public CompletionStage<JobServiceModel> scheduleDeviceMethodAsync(
-        String jobId,
-        String queryCondition,
-        MethodParameterServiceModel parameter,
-        Date startTime,
-        long maxExecutionTimeInSeconds)
-        throws ExternalDependencyException {
+            String jobId,
+            String queryCondition,
+            MethodParameterServiceModel parameter,
+            Date startTime,
+            long maxExecutionTimeInSeconds)
+            throws ExternalDependencyException, InvalidInputException {
+
+        Map<String, Object> mapPayload = new Hashtable<String, Object>();
```

Comment:
can you add a comment explaining why this is needed?

Reply:
done